sdk/src/main/java/co/omise/android/models/Capability.kt

```diff
@@ -66,3 +66,18 @@ data class Capability(
         }
     }
 }
+
```

Comment:
General point about `/capability`.  The latest version of our API looks like this (e.g. has a `payment_methods` attribute), but previous versions are incompatible (e.g. they have a `payment_backends` attribute).  The data structure is also different.

Reply:
Oh, and I guess we are setting the API version to be 2019-05-29 so it's OK 👍

Reply:
Correct. 👍